_assets/js/laws-regs-parser.js

```diff
@@ -1,4 +1,8 @@
 export default function parseLawsAndRegs (mainContent) {
+    const location = window.location.href;
+    if (location.includes('SVGAnimatedString')) {
+        history.back();
```

Comment:
does this block until after the results of going back are definitely done? (navigation like `window.location.reload()`, etc, doesn't always prevent the JS from continuing to run)

If not, maybe add a block after such as:

```
if (the SVGAnimatedString is still visible / preventing what we want) {
  setTimeout(() => parseLawsAndRegs(mainContent), 500);
  return;
}
```